### PR TITLE
Write fluvio-cloud getting-started doc

### PR DESCRIPTION
**Pull request by nicholastmosher**

### CODE REVIEW

File: content/docs/getting-started/fluvio-cloud.md [hunk 1]
```diff
@@ -1,7 +1,95 @@
...
+The first thing we need to do is create a Fluvio topic. A Topic is like a category
+where related events live together. We can create a new topic with the following
+command:
+
+```bash
+$ fluvio topic create greetings
+topic "greetings" created
+```
+
+Now that we hvae a topic, we can produce some messages! Use the following command
```
Comment (Ryanmtate) [inline content/docs/getting-started/fluvio-cloud.md:71]: hvae => have

File: content/docs/getting-started/fluvio-cloud.md [hunk 2]
```diff
@@ -1,7 +1,95 @@
...
+we'll be using that to connect to your account and download your connection
+profile.
+
+[Fluvio CLI]: ../fluvio-cli
+
+## Creating a Fluvio Cloud account
+
+Head on over to the [Fluvio Cloud signup page] to create an account.
+
+[Fluvio Cloud signup page]: https://cloud-demo.fluvio.io/signup
```
Comment (ajhunyady) [inline content/docs/getting-started/fluvio-cloud.md:24]: sound point to /singup instead.

**Comment by sehz** [comment]
Remove ref to demo cluster.

**Comment by nicholastmosher** [comment]
Anything left to do here?

**Pull request merged**